### PR TITLE
hot-fix: fix the index.html loading failed

### PR DESCRIPTION
**Pull request by sherlockvn**

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -64,46 +64,97 @@
...
+      try {
+        console.log('Loading olm.js...');
+        const olmScript = document.createElement('script');
+        olmScript.src = "assets/assets/js/package/olm.js";
+        olmScript.onload = function() {
+          console.log('olm.js loaded successfully.');
+        };
+        olmScript.onerror = function() {
+          console.error('Failed to load olm.js.');
+        };
```
Comment (hoangdat) [inline web/index.html:122]: no need this kind of logs

File: web/index.html [hunk 2]
```diff
@@ -64,46 +64,97 @@
...
+      try {
+        console.log('Loading pica.js...');
+        const picaScript = document.createElement('script');
+        picaScript.src = "https://unpkg.com/pica/dist/pica.min.js";
+        picaScript.onload = function() {
+          console.log('pica.js loaded successfully.');
+        };
+        picaScript.onerror = function() {
+          console.error('Failed to load pica.js.');
+        };
```
Comment (hoangdat) [inline web/index.html:137]: plese remove this also

File: web/index.html [hunk 3]
```diff
@@ -64,46 +64,97 @@
...
+        console.log('Loading web_support.js...');
+        const webSupportScript = document.createElement('script');
+        webSupportScript.src = "$FLUTTER_BASE_HREF/assets/packages/flutter_inappwebview_web/assets/web/web_support.js";
+        webSupportScript.defer = true;
+        webSupportScript.onload = function() {
+          console.log('web_support.js loaded successfully.');
+        };
+        webSupportScript.onerror = function() {
+          console.error('Failed to load web_support.js.');
+        };
```
Comment (hoangdat) [inline web/index.html:153]: idem

**Review by nqhhdev — APPROVED** [review]
Works well for me

**Review by hoangdat — APPROVED** [review]

**Pull request merged**